src/layer.h

```diff
@@ -127,6 +127,8 @@ class Layer
 public:
     // layer type index
     int typeindex;
+    // custom layer creator index
+    int custom_layer_creator_index;
```

Comment:
what about using (ncnn::LayerType::CustomBit | index) for typeindex to indicate the custom index ?